.github/workflows/frontend.yml

```
@@ -35,6 +35,16 @@ jobs:
         integration
         dockerfile
       codecov_flags: dockerfile-frontend
+      tags: |
+        ''
+        nydus
+      includes: |
+        - tags: nydus
+          pkg: ./frontend/dockerfile
+          kinds: |
+            integration
```

```suggestion
          kinds: |
            integration
            dockerfile
```

`dockerfile` not needed?